mlkem/native/aarch64/ntt_clean.S

```diff
@@ -29,10 +29,20 @@
 // Needed to provide ASM_LOAD directive
 #include "common.i"
 
+// Bounds:
+// In: |src| < q * C
+// Out: |dst| < q * (0.0508 * C + 1/2)
```

Comment:
As above - the 0.0508 needs a bit of explanation.

Reply:
This is explained in `reduce.c` and empirically checked in `test_bounds.py` -- I'll add a reference

Reply:
Thanks

Reply:
Thanks